src/fides/api/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -518,6 +520,11 @@ def _get_or_create_provided_identity(
                     "encrypted_value": {"value": identity_data.phone_number},
                 },
             )
+    elif target_identity_type == ProvidedIdentityType.fides_user_device_id.value:
+        identity = get_or_create_fides_user_device_id_provided_identity(
```

Comment:
Our workflows overlap more than I thought.  My workflow also uses this endpoint but will need to be able to save preferences with a  fides user device id _and_ an email/phone_number.

I think I can handle the fact that ConsentRequests are still restricted to being against one type of ProvidedIdentity downstream in my new endpoints, reticketing here:https://github.com/ethyca/fidesplus/issues/845